### DISCUSSION TITLE
Installed a /rag extension, got a wasm file, how to access it now❓

### DISCUSSION THREAD

**Discussion by fredkzk** [zed-industries/zed#21711] [Feature Requests]
I plan to make it public but I can't even see it in the extensions list. I use Win 11.
I installed the `slash-command-example` extension, same issue, I can't access it.
In both cases, I see the wasm files next to the cargo and extension files. The Zed log shows a successful compilation in both cases.
Help me please! I'd like to contribute more but I'm stuck now.
Happy holidays 🎄

**Comment by fredkzk** [comment]
Anyone wants to test it: https://github.com/fredkzk/zed-extension-rag-command
...and share some insights.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
